### PR TITLE
feat: add builder sass/less docs

### PR DESCRIPTION
**Pull request by sanyuan0704**
# PR Details

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/builder/builder-doc/en/config/tools/autoprefixer.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+
+#### Function
+
+When `tools.autoprefixer` is of Function type, the default config is passed as the first parameter and can be directly modified or returned as the final result. For example:
+
+```js
+export default {
+  tools: {
+    autoprefixer(config) {
+      // 修改 flexbox 的配置
```
Comment (chenjiahan) [inline packages/builder/builder-doc/en/config/tools/autoprefixer.md:39]: 有个中文

File: packages/builder/builder-doc/zh/config/tools/autoprefixer.md [hunk 2]
```diff
@@ -1 +1,44 @@
-TODO
+- Type: `Object` | `Function`
+- Default
+
+```js
+{
+  flexbox: 'no-2009',
+  // browserslist 取决于项目 package.json 中的 browserslist 配置及 `output.overrideBrowserslist`(优先级更高) 配置
```
Comment (chenjiahan) [inline packages/builder/builder-doc/zh/config/tools/autoprefixer.md:7]: 这样描述的话，漏了 `.browserslistrc` 的情况

可以考虑改成 `项目 package.json 中的 browserslist 配置` -> `项目中的 browserslist 配置`

File: packages/builder/builder-doc/en/config/tools/autoprefixer.md (hunk 1 above)
Comment (sanyuan0704) [reply packages/builder/builder-doc/en/config/tools/autoprefixer.md:39]: ok

File: packages/builder/builder-doc/zh/config/tools/autoprefixer.md (hunk 2 above)
Comment (sanyuan0704) [reply packages/builder/builder-doc/zh/config/tools/autoprefixer.md:7]: 嗯嗯

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**